modules/server/src/main/scala/navigate/server/tcs/TrackingConfig.scala

```diff
@@ -0,0 +1,11 @@
+// Copyright (c) 2016-2023 Association of Universities for Research in Astronomy, Inc. (AURA)
+// For license information see LICENSE or https://opensource.org/licenses/BSD-3-Clause
+
+package navigate.server.tcs
+
+case class TrackingConfig(
+  nodAchopA: Boolean,
```

Comment:
Would it help to use opaque types (NewType) to reduce the possibility of flipping parameters?